submodules/lev/.github/workflows/workflow.yml

```diff
@@ -15,7 +15,7 @@ jobs:
           - ubuntu-latest
           - windows-latest
         ocaml-compiler:
-          - 5.0.x
+          - 4.14.x
```

Comment:
I'm surprised about this change. We should be testing on 4.14 and 5.0. Could you add an appropriate change in lev for this as well?

Reply:
https://github.com/rgrinberg/lev/pull/87

I expect something like this?